src/services/request.service.js

```diff
@@ -73,6 +74,13 @@ const RequestService = {
       where: { UserId: userId },
       include: { model: db.Trips, attributes: { exclude: ['RequestId'] } } });
     return result;
+  },
+  async getTraveledDestinations() {
+    // eslint-disable-next-line no-return-await
+    return await db.Trips.findAll({ attributes: ['country', 'city', [Sequelize.fn('COUNT', 'city'), 'N of visitors']],
+      group: ['city', 'country'],
+      order: [[Sequelize.fn('COUNT', 'city'), 'DESC']]
+    });
```

Comment:
Great job @Crispy-lang. However, in this screenshot, the value on position `0`and value on position `3` has the same country and same city.

<img width="365" alt="Screen Shot 2019-11-15 at 00 17 14" src="https://user-images.githubusercontent.com/48722615/68901198-23e81300-073e-11ea-8f3f-5615ad28030f.png">